src/core/raster/qgsrasterdataprovider.cpp

```diff
@@ -800,6 +800,119 @@ QString QgsRasterDataProvider::colorInterpretationName( int bandNo ) const
   return colorName( colorInterpretation( bandNo ) );
 }
 
+QString QgsRasterDataProvider::colorName( Qgis::RasterColorInterpretation colorInterpretation ) const
+{
+  // Modified copy from GDAL
+  switch ( colorInterpretation )
+  {
+    case Qgis::RasterColorInterpretation::Undefined:
+      return QStringLiteral( "Undefined" );
+
+    case Qgis::RasterColorInterpretation::GrayIndex:
+      return QStringLiteral( "Gray" );
```

Comment:
Is there any reason why these aren't translated?

Reply:
> Is there any reason why these aren't translated?

I was wondering too. colorInterpretationName() doesn't appear to be much used, except in QgsRasterInterface::displayBandName(), so it might be save to translate it